jre_emul/Classes/com/google/j2objc/net/IosHttpURLConnection.java

```diff
@@ -545,6 +549,42 @@ private static IOException secureConnectionException(String description) {
       completionHandler(nil);
     }
   }]-*/
+  
+
+  protected List<Certificate> certificates = new ArrayList<Certificate>();
+  private static CertificateFactory certificateFactory = null;
```

Comment:
Don't make static -- Engine.getInstance() caches results, so there's no need to also cache it here.